minimessage/src/main/java/io/github/almightysatan/slams/minimessage/MMStringEntry.java

```diff
@@ -29,8 +29,21 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+/**
+ * {@link MMStringEntry}s represent a {@link String} in a language file parsed in MiniMessage format.
```

Comment:
```suggestion
 * Represents a message parsed in MiniMessage format.
```
I also really don't like the naming of this class. We should probably change it.